app/assets/stylesheets/ama_layout/layout_components/forms.scss

```diff
@@ -9,26 +9,18 @@
   float: left;
 }
 
-/* ------------- BEM Radio Buttons ------------- */
-.radio--label{
```

Comment:
It looks like forms 2 is using this class, so you will have to change the class in forms 2 when you bump the gem version in there